src/views/publish/preview.ejs

```diff
@@ -186,6 +186,6 @@
         letter-spacing: 0.08em;
         line-height: 20px;
         text-transform: uppercase;
-    }
+    } */
```

Comment:
If this is all redundant, can we bin it off?

Reply:
good spot, yeah I think it can do, table renders ok